engine/uState.mli

```diff
@@ -92,6 +92,12 @@ val emit_side_effects : Safe_typing.private_constants -> t -> t
 
 val new_univ_variable : ?loc:Loc.t -> rigid -> string option -> t -> t * Univ.Level.t
 val add_global_univ : t -> Univ.Level.t -> t
+
+(** [make_flexible_variable g a l]
+
+  Turn the variable [l] flexible, and algebraic if [a] is true and [l] can be.
+  That is if there are no strict upper constraints on [l] and
+  and it does not appear in the instance of any non-algebraic universe. *)
 val make_flexible_variable : t -> bool -> Univ.Level.t -> t
```

Comment:
The boolean should have a label here. The comment does not say explicitly what happens if `a` is `true` and `l` cannot be. Does it fail?

Also, the comment says nothing about [g].